cms/src/test/java/uk/nhs/digital/apispecs/CommonmarkMarkdownConverterTest.java

```diff
@@ -141,66 +121,25 @@ public void rendersHeadingIds_addingSuffixes_whenDuplicateHeadingValues() {
     }
 
     @Test
-    @UseDataProvider("headingsLevels")
-    public void rendersHeadings_withHeadingsHierarchyLevelsAdjustedViaParameter(
-        final int targetTopHeadingLevel,
-        final Levels initialHeadingLevels,
-        final Levels expectedHeadingLevels
-    ) {
+    public void rendersHeadings_withHeadingsHierarchyLevelsAdjustedViaParameter() {
         // given
-        final String markdown = markdownWithHeadingsAt(initialHeadingLevels);
+        final String markdown = from("headingsWithHierarchyLevels.md");
 
-        final String expectedHtml = htmlWithHeadingsAt(expectedHeadingLevels);
+        final String expectedHtml = from("headingsWithHierarchyLevels.html");
```

Comment:
i'm guessing you've made the move to a static test file to get around issues with \<hr\> elements throwing off your assertions - i think this makes the test weaker since it now covers much less cases

you could always strip the hr elements from the text before comparing them, similarly to how we strip whitespace in other methods? and write a test method specifically for checking that hr's are inserted where we expect them